.github/workflows/conformance-tetragon-gke.yaml

```diff
@@ -42,7 +42,10 @@ jobs:
         id: 'auth'
         uses: 'google-github-actions/auth@140bb5113ffb6b65a7e9b937a81fa96cf5064462'
         with:
-          credentials_json: '${{ secrets.GCP_PR_SA_KEY }}'
+          workload_identity_provider: ${{ secrets.GCP_PR_WORKLOAD_IDENTITY_PROVIDER }}
+          service_account: ${{ secrets.GCP_PR_SA }}
```

Comment:
I updated these organization secrets to be accessible from cilium/charts, so that should hopefully work. cc @cilium/ci-structure @cilium/github-sec

Reply:
I've reverted this as the whole approach was changed in https://github.com/cilium/charts/pull/187 .